app/templates/list.hbs

```diff
@@ -33,15 +39,25 @@
   </table.menu>
   <table.content as |c|>
     <c.header>
-      <th><span class="au-c-data-table__header-title au-c-data-table__header-title--sortable">{{t 'reglementList.title'}}</span></th>
+      <AuDataTableThSortable @field="document.currentVersion.title" @currentSorting={{this.sort}} @label={{t 'reglementList.title'}} />
       <AuDataTableThSortable @field="document.currentVersion.createdOn" @currentSorting={{this.sort}} @label={{t 'reglementList.createdOn'}} />
+      <AuDataTableThSortable @field="document.currentVersion.updatedOn" @currentSorting={{this.sort}} @label={{t 'reglementList.updatedOn'}} />
+      <AuDataTableThSortable @field="document.currentVersion.title" @currentSorting={{this.sort}} @label={{t 'reglementList.publishDate'}} />
```

Comment:
The field property should probably be different here, but this too will change with the new model soon.

Reply:
@elpoelma ok, will wait till then. Please let me know what should be the field property after we have new model